src/Chapter21/Listing21.05.CancelingParallelLoop.cs

```diff
@@ -25,23 +28,25 @@ public static void Main()
                directoryPath, searchPattern,
                SearchOption.AllDirectories);
 
+#region HIGHLIGHT
             CancellationTokenSource cts =
```

Comment:
```suggestion

            #region HIGHLIGHT
            CancellationTokenSource cts =
```
nit: make the indentation the same